elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -711,7 +711,9 @@ elytron.mapped-role-mapper.remove=The remove operation for the role mapper.
 # Attributes
 elytron.mapped-role-mapper.keep-mapped=When set to 'true' the mapped roles will retain all roles, that have defined mappings.
 elytron.mapped-role-mapper.keep-non-mapped=When set to 'true' the mapped roles will retain all roles, that have no defined mappings.
-elytron.mapped-role-mapper.role-map=A string to string list map for mapping roles.
+elytron.mapped-role-mapper.role-map=A list of mappings to use for mapping roles.
+elytron.mapped-role-mapper.role-map.from=The role to match for mapping roles.
+elytron.mapped-role-mapper.role-map.to=A string list of roles to map to.
```

Comment:
Just a note for other reviewers, these are not new elements, they were present in the XML schema before as well. 

This PR looks great to me. CLI command changed but old format continues to work, so I don't think RFE process is needed, but will let @fjuma to confirm. @cam-rod LGTM